.buildkite/setup.sh

```diff
@@ -6,4 +6,5 @@ module load lc
 module load syn/latest
 module load genus
 module load innovus/19.10.000
+module load prime
```

Comment:
Sometimes /latest isn't set up properly so it's better to specify the exact version you want to use.

Reply:
prime/T-2022.03

Reply:
> prime/T-2022.03

done